lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -24,12 +24,12 @@
     color: #444;
   }
 
-  .slideshow__slide .container > .wrapper {
-    margin: 12px;
+  .wrapper.-compact {
```

Comment:
Can you switch this to `.container.-compact > .wrapper` like suggested in my original comment? We can't target modifiers on the wrapper itself because that's a common classname used in multiple components.